src/main/frontend/fs.cljs

```diff
@@ -194,14 +194,12 @@
 
 (defn mkdir-if-not-exists
   [dir]
-  (->
-   (when dir
-     (util/p-handle
-      (stat dir)
-      (fn [_stat])
-      (fn [_error]
-        (mkdir! dir))))
-   (p/catch (fn [error] (js/console.error error)))))
+  (when dir
+    (util/p-handle
+     (stat dir)
+     (fn [_stat])
+     (fn [_error]
+       (mkdir! dir)))))
 
 ;; FIXME: counterintuitive return value
```

Comment:
Not sure what this means.

Reply:
> return false on written, true on already exists

The primary verb of the fn name should be `create`, not `exists`.